projects/step-frontend/src/lib/modules/timeseries/components/chart-dashlet/chart-dashlet.component.html

```diff
@@ -13,16 +13,26 @@
   }
 </div>
 
-@if (showHigherResolutionWarning) {
-  <div class="resolution-warning">
-    <step-icon
-      name="alert-triangle"
-      matTooltip="Data for requested resolution doesn't exist. Used higher resolution of <{{
-        resolutionLabels[collectionResolutionUsed.toString()] || collectionResolutionUsed
-      }}> instead."
-    />
-  </div>
-}
+<div class="warnings-container">
+  @if (showMissingTTLWarning) {
+    <div>
+      <step-icon
+        name="alert-triangle"
+        matTooltip="Requested range is out of the current TTL (time to live) configuration"
```

Comment:
```suggestion
        matTooltip="Requested range is out of the configured TTL (Time To Live) in the Housekeeping settings"
```

I would like to let users know where this was set